### PR TITLE
fix: replace treeShake with globalImports

### PR DESCRIPTION
**Pull request by husayt**
treeShake has been renamed to globalImports everywhere except for plugin.js This is fixed with this pull request

### CODE REVIEW

**Comment by kevinmarrec** [comment]
nice cathc @husayt , but seems there's something to do as well around plugin registration : https://github.com/nuxt-community/vuetify-module/blob/next/src/plugin.ts#L23 , `globalImports` should be given instead of `treeShake`

**Comment by husayt** [comment]
@kevinmarrec this is done now

File: src/plugin.ts [hunk 1]
```diff
@@ -26,8 +26,7 @@ export default function setupPlugin (this: ModuleThis, options: Options) {
     src: path.resolve(__dirname, '../templates', 'plugin.js'),
     options: {
       defaultIconPreset: options.defaultAssets && options.defaultAssets.icons,
-      preset: options.preset,
-      treeShake: true
+      preset: options.preset
```
Comment (kevinmarrec) [inline src/plugin.ts]:
```suggestion
      globalImports: options.globalImports,
      preset: options.preset
```

File: templates/plugin.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import Vue from 'vue'
-import Vuetify from '<%= options.treeShake ? 'vuetify/lib' : 'vuetify' %>'
+import Vue from 'vuetify/lib'
```
Comment (kevinmarrec) [inline templates/plugin.js]: 🤷‍♂️
```suggestion
import Vuetify from 'vuetify/lib'
```

Comment (husayt) [reply templates/plugin.js]: Seems you requested this while I was doing this change.

**Comment by husayt** [comment]
@kevinmarrec seems you were looking at old changes. Can you check again pls

**Comment by kevinmarrec** [comment]
@husayt  Still doesn't look good to me in "Files" tab 🤷, is it GH issue or ?

**Comment by husayt** [comment]
Can you point to what is wrong @kevinmarrec . I actually did those changes directly in github as I didn't have a chance to edit files locally. Feel free to change what doesn't look right.

**Comment by kevinmarrec** [comment]
Well the things you marked as resolved are not resolved :
- `Vuetify` instead of `Vue`
- Missing `globalImports` when given options to plugin

**Comment by husayt** [comment]
Indeed, you are right. Something is weird with Github editor, I am sure I did those changes yesterday. Nevermind, all should be fine now.

**Comment by kevinmarrec** [comment]
I will release later this day :)

**Pull request merged**